### PR TITLE
SOV-3136: update lend page text

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-3136

### CODE REVIEW

File: apps/frontend/src/app/5_pages/LendPage/components/AdjustModal/LendingForm.tsx [hunk 1]
```diff
@@ -164,8 +164,13 @@ export const LendingForm: FC<DepositProps> = ({ state, onConfirm }) => {
       )}
       <SimpleTable className="mt-8">
         <SimpleTableRow
-          label={t(translations.lendingAdjust.newApy)}
-          value={<AmountRenderer value={newApy} suffix="% APY" />}
+          label={t(translations.lendingAdjust.newApr)}
+          value={
+            <AmountRenderer
+              value={newApy}
+              suffix={`% ${t(translations.lending.apr)}`}
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/AdjustModal/LendingForm.tsx]: This looks really weird, pls verify with @john-light if we can display the value without the `LEND APR` suffix.
image

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/LendFrame.constants.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const COLUMNS_CONFIG = [
     id: 'lendApy',
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/LendFrame.constants.tsx]: Adjust the `id` as well.

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx [hunk 3]
```diff
@@ -89,7 +89,7 @@ export const getChartData = (
     labels: mockData.dates,
     datasets: [
       {
-        label: t(translations.lendPage.table.lendApy),
+        label: t(translations.lendPage.table.lendApr),
         data: mockData.lendAPY,
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx]: Adjust `mockData`.

File: apps/frontend/src/app/5_pages/LendPage/components/LendModal/CurrentStats.tsx [hunk 4]
```diff
@@ -31,7 +31,7 @@ export const CurrentStats: FC<CurrentStatsProps> = ({ apy, token }) => {
       <div className="flex gap-8">
         <div className="mt-6 flex flex-col gap-2">
           <Paragraph className="font-medium text-gray-30">
-            {t(translations.lending.apy)}
+            {t(translations.lending.apr)}
           </Paragraph>
           <AmountRenderer value={apy} suffix="%" />
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendModal/CurrentStats.tsx]: Adjust the prop name as well.

File: apps/frontend/src/app/5_pages/LendPage/components/LendModal/LendingForm.tsx [hunk 5]
```diff
@@ -113,8 +113,13 @@ export const LendingForm: FC<DepositProps> = ({ state, onConfirm }) => {
       )}
       <SimpleTable className="mt-6">
         <SimpleTableRow
-          label={t(translations.lending.newApy)}
-          value={<AmountRenderer value={lendApy} suffix="% APY" />}
+          label={t(translations.lending.newApr)}
+          value={
+            <AmountRenderer
+              value={lendApy}
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendModal/LendingForm.tsx]: Adjust `lendApy`.

File: apps/frontend/src/locales/en/translations.json [hunk 6]
```diff
@@ -797,8 +797,8 @@
         "table": {
             "lend": "Lend",
             "asset": "Asset",
-            "lendApy": "Lend APY",
-            "lendApyMobile": "APY",
+            "lendApr": "Lend APR",
+            "lendApyMobile": "APR",
             "lendApyInfo": "Interest rate is variable based on the current utilization rate and loans outstanding",
```
Comment (tiltom) [inline apps/frontend/src/locales/en/translations.json]:
```suggestion
            "lendAprMobile": "APR",
            "lendAprInfo": "Interest rate is variable based on the current utilization rate and loans outstanding",
```

**Review by tiltom — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/2_molecules/FixedInterestPoolRowTitle/FixedInterestPoolRowTitle.tsx [hunk 7]
```diff
@@ -33,7 +33,7 @@ export const FixedInterestPoolRowTitle: FC<FixedInterestPoolRowTitleProps> = ({
       <span className="text-gray-30 ml-1 font-medium">
         {isBorrow
```
Comment (tiltom) [inline apps/frontend/src/app/2_molecules/FixedInterestPoolRowTitle/FixedInterestPoolRowTitle.tsx:34]: @john-light - this condition was there to distinguish between borrow/lend and to display `APR`/`APY` accordingly. Was that incorrect and should we use `APR` now everywhere?

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.tsx [hunk 8]
```diff
@@ -18,14 +18,14 @@ export const LendFrameChart: FC<LendFrameProps> = memo(({ pool }) => {
   const { lendHistory } = useGetLendHistory(asset);
 
   const mockData = convertPoolHistoryToMockData(lendHistory);
-  const lendAPYTickStep = useMemo(
-    () => (Math.max(...mockData.lendAPY) - Math.min(...mockData.lendAPY)) / 6,
-    [mockData.lendAPY],
+  const lendAPRTickStep = useMemo(
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.tsx]:
```suggestion
  const lendAprTickStep = useMemo(
```

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.types.ts [hunk 9]
```diff
@@ -1,6 +1,6 @@
 export type ChartData = {
   date: Date;
-  lendAPY: number;
+  lendAPR: number;
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.types.ts]:
```suggestion
  lendApr: number;
```
Make sure that you use our naming conventions everywhere - capitalize the first letter only.

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx [hunk 10]
```diff
@@ -58,12 +58,12 @@ export const convertPoolHistoryToMockData = (
   const dates = poolHistory.map(entry =>
     dayjs(entry.timestamp).format('YYYY-MM-DD'),
   );
-  const lendAPY = poolHistory.map(entry => parseFloat(entry.supply_apr));
+  const lendAPR = poolHistory.map(entry => parseFloat(entry.supply_apr));
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx]:
```suggestion
  const lendApr = poolHistory.map(entry => parseFloat(entry.supply_apr));
```

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx [hunk 11]
```diff
@@ -111,7 +111,7 @@ export const getChartData = (
 };
 
 export const getChartOptions = (
-  lendAPYTickStep: number,
+  lendAPRTickStep: number,
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx]:
```suggestion
  lendAprTickStep: number,
```

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx [hunk 12]
```diff
@@ -123,22 +123,22 @@ export const getChartOptions = (
   const fontWeight = '500';
   const dashGridType = 'dash';
 
-  const lendAPYAxisOptions = {
+  const lendAPRAxisOptions = {
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.utils.tsx]:
```suggestion
  const lendAprAxisOptions = {
```

File: apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/hooks/useGetLendFrameChartOptions.ts [hunk 13]
```diff
@@ -1,8 +1,11 @@
+import { t } from 'i18next';
+
+import { translations } from '../../../../../../../../locales/i18n';
 import { LendingPool } from '../../../../../../../../utils/LendingPool';
 import { MockData } from '../LendFrameChart.types';
 
 export const useGetLendFrameChartOptions = (
-  lendAPYTickStep: number,
+  lendAPRTickStep: number,
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/hooks/useGetLendFrameChartOptions.ts]:
```suggestion
  lendAprTickStep: number,
```

**Review by tiltom — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/2_molecules/FixedInterestPoolRowTitle/FixedInterestPoolRowTitle.tsx (hunk 7 above)
Comment (john-light) [reply apps/frontend/src/app/2_molecules/FixedInterestPoolRowTitle/FixedInterestPoolRowTitle.tsx:34]: @tiltom  you should use APR in the places where it is specified to use APR, and APY where it is specified to use APY.

**Review by tiltom — APPROVED** [review]

**Pull request merged**